Software/bsw/static/Mcal/CAN/src/Can.c

```diff
@@ -1230,88 +1225,104 @@ Std_ReturnType Can_SetControllerMode(uint8 Controller,
 	return ret;
 }
 
-Std_ReturnType Can_GetControllerMode(uint8 Controller,
-		Can_ControllerStateType* ControllerModePtr) {
+Std_ReturnType Can_GetControllerMode(uint8 Controller,Can_ControllerStateType* ControllerModePtr) 
```

Comment:
Following AUTOSAR_SWS_C_ImplementationRules documentation is very helpful for code readability and to avoid simple errors like performing assignment in if condition rather than checking. e.g. if (NULL_PTR == ConfigPtr) {..} rather than if (ConfigPtr==NULL_PTR).

Reply:
I will work to modify the if conditions with (==) to make the value in the Left hand side of the condition.